src/patternfly/components/MenuToggle/menu-toggle.hbs

```diff
@@ -1,4 +1,4 @@
-<button class="pf-c-menu-toggle
+<div class="pf-c-menu-toggle
```

Comment:
This element should still be a button by default, just needs to be a div for this version and other versions down the road, like split button, which we'll add at some point. Typically the way we do this is via `{{#if menu-toggle--type}}{{menu-toggle--type}}{{else}}button{{/if}}` for the opening and close tags, which leaves the default as it is (doesn't break any current examples) and allows you to set `menu-toggle--type="div"` on the typeahead example. Here's the select toggle as an example - https://github.com/patternfly/patternfly/blob/main/src/patternfly/components/Select/select-toggle.hbs


```suggestion
<{{#if menu-toggle--type}}{{menu-toggle--type}}{{else}}button{{/if}} class="pf-c-menu-toggle
```